src/colors.module.scss

```
@@ -0,0 +1,7 @@
+$standard-blue: #3A4374;
+$white: #FFF;
+
+:export {
+  standard-blue: $standard-blue;
+  white: $white;
+}
```

```suggestion
}

```
All files should end with new-line character